src/nullablevector.jl

```diff
@@ -311,3 +311,80 @@ function Base.empty!(X::NullableVector)
     empty!(X.isnull)
     return X
 end
+
+function Base.promote_rule{T1,T2}(::Type{T1}, ::Type{Nullable{T2}})
+    promote_rule(Nullable{T2}, T1)
+end
+
+function Base.typed_hcat{T}(::Type{T}, A::AbstractVecOrMat...)
```

Comment:
It's not OK to override Base methods like this with signatures that only involve Base types. That will affect unrelated packages and is known as type piracy.